src/Uno.UI/UI/Xaml/UIElement.Pointers.wasm.cs

```diff
@@ -308,7 +308,8 @@ private object CoerceHitTestVisibility(object baseValue)
 			}
 
 			// If we're not locally hit-test visible, visible, or enabled, we should be collapsed. Our children will be collapsed as well.
-			if (!IsLoaded || !IsHitTestVisible || Visibility != Visibility.Visible || !IsEnabledOverride())
+			// SvgElements are an exception here since they won't be loaded.
```

Comment:
I wonder if there are other 'non-FrameworkElement UIElements' to which this might apply? @carldebilly ?

Reply:
SVG are special...  I think they are alone.  Maybe we should implement a kind of "SVGLoaded" feature instead of just bypassing it like here, but for this PR I think it's acceptable.